slm_porosity/heat_transfer.f90

```diff
@@ -497,7 +499,10 @@ SUBROUTINE user_read_input()
     call input_real ('initial_enthalpy', initial_enthalpy, 'default')
     call input_real ('conductivity_der', conductivity_der, 'stop')
     call input_real ('capacity_der', capacity_der, 'stop')
+    call input_integer ('smoothing_method', smoothing_method, 'stop')
     call input_real_vector ('x0', x0, 3, 'stop')
+    smoothing_width = (fusion_heat+fusion_delta)/6.0_pr
```

Comment:
This parameter shows which part of the liqudus-solidus width we extrapolate with the spline instead semiwise function, so there is no way how it could be used just with a default number.

Reply:
I repeat. This parameter should be independent of other input variables.